src/components/about/About.js

```diff
@@ -0,0 +1,42 @@
+import React from "react"
+import "./about.scss"
+
+const About = props => {
+  return (
+    <div className="outer-container">
+      <div className="about-container">
+        <h1>
+          <span id="about-us-header">ABOUT US</span>
+        </h1>
+        <p>
+          For entry level developers, people interested in starting a new career
+          in tech, and anyone wanting to encourage and help newer developers.
+          Join us to hear relevant speakers, have a slice of pizza, and make new
+          connections! We look forward to meeting you. If you’d like to be a
+          part of our slack channel, you can be a part of the conversation here:
+          http://bit.ly/JrDevSlackInvite
+        </p>
+        <p>
+          PDX Jr. Devs is a community of developers comprised of people that are
+          getting their start in tech
+        </p>
+        <p>
```

Comment:
This lorem ipsum does match the mock :smile: but it's not super helpful to human visitors to our site. Let's either remove it (totally fine) or put more info here (if you think there's more to be said here at this time).

Reply:
Removed lorem ipsum block